File: frontend/olympic_it_project/olympic_it_project/lib/dto/admin_manager/exam/add_participant_request.dart
class AddParticipantRequest {
  final int examId;
  final int userId;

  AddParticipantRequest({
    required this.examId,
    required this.userId,
  });

  Map<String, dynamic> toJson() {
    return {
      'examId': examId,
      'userId': userId,
    };
  }
}

File: backend/olympic_it_project/src/main/java/org/example/olympic_ot_project/dto/exam/AddParticipantRequest.java
package org.example.olympic_ot_project.dto.exam;

import jakarta.validation.constraints.NotNull;
import lombok.Data;

@Data
public class AddParticipantRequest {
    @NotNull(message = "Id đề thi không được để trống")
    private Integer examId;

    @NotNull(message = "Id thí sinh không được để trống")
    private Integer userId;

    @NotNull(message = "Số ngồi đại diện không được để trống")
    private Integer seatNumber;
}